src/CommandingOfficers/CommanderStrong.java

```diff
@@ -4,19 +4,30 @@
 import CommandingOfficers.Modifiers.CODamageModifier;
 import CommandingOfficers.Modifiers.COModifier;
 import CommandingOfficers.Modifiers.COMovementModifier;
+import Engine.Combat.CombatEngine;
+import Engine.Combat.StrongCombatModifier;
 
 public class CommanderStrong extends Commander
 {
   private static final CommanderInfo coInfo = new CommanderInfo("Strong", CommanderLibrary.CommanderEnum.STRONG);
-
+  StrongCombatModifier strongMod;
+  
   public CommanderStrong()
   {
     super(coInfo);
 
+    // TODO: Super alpha values. Need balance.
+    starsMinor = 3;
+    starsMax = 6;
+    starsCurrent = 6;
+
     // Set Cmdr Strong up with a base damage buff and long-range APCs. These COModifiers are
     // not added to the modifers collection so they will not be reverted.
-    COModifier strongMod = new CODamageModifier(20); // Give us a nice base power boost.
-    strongMod.apply(this);
+    // COModifier strongMod = new CODamageModifier(20); // Give us a nice base power boost.
+    // strongMod.apply(this);
+    strongMod = new StrongCombatModifier(this,20);
```

Comment:
OK, uh. Why didn't you just use a CombatDamageModifier here? Do we really need a new class that's specific to Strong?